### PR TITLE
Added fill_zeros_like (U)INT8/BF16/FP32 oneDNN kernel for video detection models

### PR DESCRIPTION
**Pull request by jakpiase**
### PR types
New features

### PR changes
OPs

### Describe
Added fill_zeros_like (U)INT8/BF16/FP32 oneDNN kernel for video detection models which has not been delivered yet.

### CODE REVIEW

File: paddle/fluid/operators/mkldnn/fill_zeros_like_mkldnn_op.cc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                                          dnnl::eltwise_backward>(engine,
+                                                                  cpu_place) {
+    // uint8_t is always used to treat float NaNs like normal numbers
+    auto md = dnnl::memory::desc(
+        {out->numel() * static_cast<int64_t>(sizeof(T))},
+        dnnl::memory::data_type::u8, dnnl::memory::format_tag::a);
+
+    this->AcquireForwardPrimitiveDescriptor(dnnl::prop_kind::forward_training,
+                                            dnnl::algorithm::eltwise_linear, md,
+                                            0.0f, 0.0f);
```
Comment (lidanqing-vv) [inline paddle/fluid/operators/mkldnn/fill_zeros_like_mkldnn_op.cc]: How do we know that this oneDNN kernel will be faster than native kernel? Or we have to wait for real models to check ?

Comment (jakpiase) [reply paddle/fluid/operators/mkldnn/fill_zeros_like_mkldnn_op.cc]: We have to wait and check

**Comment by lidanqing-vv** [comment]
Sorry I need to close temporarily

**Comment by jakpiase** [comment]
Closing due to low priority and the fact that video-detection models are not available

**Pull request closed without merging**